src/editor/serialize.ts

```diff
@@ -31,7 +31,8 @@ export function mdSerialize(model: EditorModel) {
                 return html + part.text;
             case "room-pill":
             case "user-pill":
-                return html + `[${part.text.replace(/[[\\\]]/g, c => "\\" + c)}](${makeGenericPermalink(part.resourceId)})`;
+                return html +
+                    `[${part.text.replace(/[[\\\]]/g, c => "\\" + c)}](${makeGenericPermalink(part.resourceId)})`;
```

Comment:
literally 1 character over :(

Reply:
Yeah but that'll keep adding up and then my viming will be very clunky and sad